ipn/ipnauth/ipnauth.go

```diff
@@ -25,6 +26,32 @@ import (
 	"tailscale.com/version/distro"
 )
 
+var ErrNotImplemented = errors.New("not implemented for GOOS=" + runtime.GOOS)
+
+type WindowsToken interface {
```

Comment:
Needs docs, especially as LocalBackend references this.

// WindowsToken represents a Windows user and their current security
// context.

?